INSTALL.md

```diff
@@ -0,0 +1,128 @@
+# System requirements and Installation
+
+## Node.js
+
+It is a node.js project. Therefore you need node.js installed. Get it from [http://nodejs.org](http://nodejs.org).
+
+You need a current 4.x version to run the software (0.12.x should also be OK).
+
+Your node.js ships a npm in suitable version.
+
+## Additional Software
+
+Some modules are compiled during the installation. Therefore some additional software must be installed.
+
+### Python
+
+* Python 2.7 (not Python 3!) in system path
+* For Mac OS: Python 2.7 is already installed.
+* For Windows:
+  * Make sure that Python and node.js are both either 32 bit or 64 bit - mixed combinations will not work!
+  * Add an environment variable `PYTHON` that points to the Python executable.
+
+### C++
```

Comment:
yes. can't remember which exactly, but emptying your node_modules and npm install from scratch would show you in the transcript